exercises/hello-world/TUTORIAL.md

```diff
@@ -3,11 +3,7 @@ https://github.com/exercism/kotlin/blob/master/exercises/hello-world/TUTORIAL.md
 
```

Comment:
Please add a `hints.md` file to direct users to this tutorial (as in https://github.com/exercism/java/blob/master/exercises/hello-world/.meta/hints.md).

Reply:
Please also port the new Introduction and Exercise Structure sections from the Java tutorial file.